log4j-1.2-api/src/main/java/org/apache/log4j/builders/AbstractBuilder.java

```diff
@@ -86,12 +86,25 @@ protected String getNameAttribute(Element element) {
         return element.getAttribute(NAME_ATTR);
     }
 
+    /**
+     * Normalized version of the "name" attribute of the &lt;param&gt; tag.
+     * @param element
+     * @return
+     */
+    protected String getNormalizedNameAttribute(Element element) {
+        return capitalize(getNameAttribute(element));
+    }
+
```

Comment:
Are all call sites of getNameAttribute() now call sites of this new method? If so, just update getNameAttribute() IMO.

Reply:
No, `getNameAttribute` is used for retrieving the `"name"` attribute of both the `<appender>` and `<param>` tags. AFAIK appender names are case sensitive.